google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -47,10 +47,52 @@ class Bucket
         attr_accessor :gapi
 
         ##
+        # A `true`/`false` value or a project ID string to be sent as the
+        # `userProject` query param for operations on requester pays buckets and
+        # their files. The default is `nil`. For convenience, this attribute
+        # should be set when first retrieving the bucket, by providing the
+        # `user_project` option to {Project#bucket}.
+        #
+        # If the `requester_pays` flag is enabled for the bucket, and if this
+        # attribute is set to `true`, transit costs for operations on the bucket
+        # will be billed to the current project for this client. (See
+        # {Project#project} for the ID of the current project.) If this
+        # attribute is set to a project ID other than the current project, and
+        # that project is authorized for the currently authenticated service
+        # account, transit costs will be billed to the given project.
+        #
+        # The requester pays feature is currently available only to whitelisted
+        # projects.
+        #
+        # See also {#requester_pays=} and {#requester_pays}.
```

Comment:
Maybe change to: See also {#requester_pays=} and {#requester_pays} to turn on requester pays for a bucket.